src/test/java/com/snowflake/kafka/connector/SinkTaskIT.java

```diff
@@ -323,4 +328,135 @@ public void testMultipleSinkTasksWithLogs() throws Exception {
     assert offsetMap1.get(topicPartitions0.get(0)).offset() == BUFFER_COUNT_RECORDS_DEFAULT;
     assert offsetMap0.get(topicPartitions1.get(0)).offset() == BUFFER_COUNT_RECORDS_DEFAULT;
   }
+
+  @Test
+  public void testTopicToTableRegex() {
+    Map<String, String> config = TestUtils.getConf();
+    SnowflakeSinkConnectorConfig.setDefaultValues(config);
+
+    testTopicToTableRegexMain(config);
+  }
+
+  // runner for topic to table regex testing, used to test both streaming and snowpipe scenarios.
```

Comment:
This is a little hacky, but I'd rather do this than copy-paste the test twice. I tried moving it to testUtils, however the scope of SnowflakeSinkConnectorConfig.TOPICS_TABLES_MAP and a couple other variables were not accessible. 

Any thoughts on how to improve this?